x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/query_tab.cy.ts

```diff
@@ -14,64 +14,54 @@ import {
   TIMELINE_QUERY,
   NOTE_CARD_CONTENT,
 } from '../../../screens/timeline';
+import { deleteTimelines } from '../../../tasks/api_calls/common';
+import { addNoteToTimeline } from '../../../tasks/api_calls/notes';
 import { createTimeline } from '../../../tasks/api_calls/timelines';
 
 import { login } from '../../../tasks/login';
 import { visit } from '../../../tasks/navigation';
 import {
   addFilter,
+  addNoteToFirstRowEvent,
   openTimelineById,
   pinFirstEvent,
   refreshTimelinesUntilTimeLinePresent,
 } from '../../../tasks/timeline';
 
 import { TIMELINES_URL } from '../../../urls/navigation';
 
-describe.skip('Timeline query tab', { tags: ['@ess', '@serverless'] }, () => {
-  before(() => {
+const defaultTimeline = getTimeline();
+
+describe('Timeline query tab', { tags: ['@ess', '@serverless'] }, () => {
+  beforeEach(() => {
     login();
     visit(TIMELINES_URL);
-    createTimeline(getTimeline())
+    deleteTimelines();
+    createTimeline(defaultTimeline)
       .then((response) => response.body.data.persistTimeline.timeline.savedObjectId)
       .then((timelineId: string) => {
-        refreshTimelinesUntilTimeLinePresent(timelineId)
-          // This cy.wait is here because we cannot do a pipe on a timeline as that will introduce multiple URL
-          // request responses and indeterminism since on clicks to activates URL's.
-          .then(() => cy.wrap(timelineId).as('timelineId'))
-          // eslint-disable-next-line cypress/no-unnecessary-waiting
-          .then(() => cy.wait(1000))
-          // TO-DO: Issue 163398
-          // .then(() =>
-          //   addNoteToTimeline(getTimeline().notes, timelineId).should((response) =>
-          //     expect(response.status).to.equal(200)
-          //   )
-          // )
-          .then(() => openTimelineById(timelineId))
-          .then(() => pinFirstEvent())
-          // TO-DO: Issue 163398
-          // .then(() => persistNoteToFirstEvent('event note'))
-          .then(() => addFilter(getTimeline().filter));
+        cy.wrap(timelineId).as('timelineId');
+        refreshTimelinesUntilTimeLinePresent(timelineId);
+        addNoteToTimeline(defaultTimeline.notes, timelineId);
+        openTimelineById(timelineId);
+        pinFirstEvent();
+        addFilter(defaultTimeline.filter);
       });
   });
 
   describe('Query tab', () => {
```

Comment:
this describe does absolutely nothing, can we remove it?

Reply:
also, these 4 tests should definitely be merged into 1. You could argue that shouldn't even be Cypress tests but I feel that until we have component testing there is value in keeping them.
Nothing should prevent us from merging them into a single test though

Reply:
Done and definitely agree. For now just going to clean this up according to the `tasks` changes required, and do a separate PR focused on migrating certain tests to jest. This can definitely be one of them